src/06-mocking-node-api/index.test.ts

```diff
@@ -29,24 +39,36 @@ describe('doStuffByInterval', () => {
   });
 
   test('should set interval with provided callback and timeout', () => {
-    // Write your test here
+    const func = jest.fn();
+    const spy = jest.spyOn(global, 'setInterval');
+    doStuffByInterval(func, 100);
+    expect(spy).toHaveBeenCalledWith(func, 100);
   });
 
   test('should call callback multiple times after multiple intervals', () => {
-    // Write your test here
+    const func = jest.fn();
+    const INTERVAL = 100;
+    doStuffByInterval(func, INTERVAL);
+    expect(func).not.toBeCalled();
+    jest.advanceTimersByTime(INTERVAL * 5);
+    expect(func).toHaveBeenCalledTimes(5);
   });
 });
 
 describe('readFileAsynchronously', () => {
   test('should call join with pathToFile', async () => {
-    // Write your test here
+    const pathToFile = '../../package.json';
+    const spy = jest.spyOn(path, 'join');
+    const readFile = async () => await readFileAsynchronously(pathToFile);
+    readFile();
+    expect(spy).toHaveBeenCalledWith(__dirname, pathToFile);
   });
 
   test('should return null if file does not exist', async () => {
-    // Write your test here
+    expect(await readFileAsynchronously('.ggwp')).toBeNull();
```

Comment:
this is not correct implementation of test. You shouldnt use real filesystem in test. And your test dont guaranteed that the file doesnt exist.
You need to mock fs.existsSync  function and imitate that it return false.